server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -202,7 +203,7 @@ while read size result ;do
         continue
     fi
 
-    # create the link from results to incoming *before* moving the directory from TMP.
+    # create the link from results to incoming *before* moving the directory from UNPACK_PATH.
     # That avoids a race with pbench-create-results-links which might create a recursive link
```

Comment:
Do we still run pbench-create-results-links? If not, we should get rid of the comment.